### ISSUE TITLE
<fix>type error in TrainService.train

### ISSUE THREAD

**Issue by bigtrader91** [gume-gume/MLOps-project#51]
### ✔코드경로(line 포함)
- path: service/app_service.py
line: 118

### ✔ 오류메시지
`For evaluating multiple scores, use sklearn.model_selection.cross_validate instead. {'accuracy'} was passed.
`

### ✔ 버전정보
python 3.9.7

### ✔ 버그에 대한 뇌피셜
- rf_optimization 메소드에서  n_splits, n_trials ,accuracy 타입을 찍어봤더니, set 으로 나오면서 발생하는 에러로 생각됩니다
파이단틱 통해서 각각 int, int, str로 타입을 지정했는데 메소드에서 인자로 받아들일 때에는 set타입으로 와서 
그 중간 과정에서 타입변경이 일어나는 것 같습니다.

### ✔  하고싶은 말(optional)
- rf_optimization 메소드에서 타입 변경하는 로직을 넣어도 되지만, set으로 바뀌는 원인을 파악해야할 듯 합니다.
